code-scanning/sobelow.yml

```diff
@@ -0,0 +1,38 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
```

Comment:
Please add some comments about what this workflow does and if there are any pre-requisites.
Also, add links for user to read more or get support.